arch/avr/src/at32uc3/at32uc3_gpio.c

```diff
@@ -63,19 +63,19 @@
 static uint32_t g_portmap[AVR32_NGPIO_PORTS] =
 {
 #if AVR32_NGPIO > 0
-  AVR32_GPIO0_BASE
+  AVR32_GPIO0_BASE,
```

Comment:
why need change

Reply:
I think it is only aesthetic, but the way he suggests seems better to visualize